src/main/java/org/alfresco/repo/content/transform/LegacyTransformerDebug.java

```diff
@@ -496,11 +497,11 @@ public int compare(ContentTransformer transformer1, ContentTransformer transform
                                 ? false
                                 : remoteTransformServiceRegistry.isSupported(sourceMimetype,
                                 -1, targetMimetype, Collections.emptyMap(), null);
-                        LocalTransform localTransform = localTransformServiceRegistryImpl == null
-                                ? null
-                                : localTransformServiceRegistryImpl.getLocalTransform(sourceMimetype,
-                                -1, targetMimetype, Collections.emptyMap(), null);
-                        if (localTransform != null || supportedByTransformService || size >= 1)
+                        List<SupportedTransform> localTransformers = localTransformServiceRegistryImpl == null
```

Comment:
The code changes in this class are very similar to those in the super class. The two classes repeat much of the same code rather than having common code in the super class as the plan is to simply remove this class shortly. As a result the super class remains simpler to understand.